pkg/controller/storagecluster/util.go

```
@@ -124,3 +126,10 @@ func isEmpty(object interface{}) bool {
 	}
 	return false
 }
+
+func forceContinueUpgrade(
+	cluster *corev1.StorageCluster,
+) bool {
+	force, err := strconv.ParseBool(cluster.Annotations[constants.AnnotationForceContinueUpdate])
+	return err == nil && force
```

please write a log if err is not nil as it is swallowed